api/BUILD

```diff
@@ -117,19 +88,64 @@ proto_library(
         "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
         "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
         "//envoy/config/retry/previous_priorities:pkg",
+        "//envoy/config/route/v3alpha:pkg",
```

Comment:
It would be great to have a top-level README.md that articulates the philosophy of the namespaces. i know we have some scattered README.md, e.g. https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/README.md, but it would be nice to know when adding new config, where it should logically go.